apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -205,23 +215,25 @@ maybe_add_overlay_to_lashup(Overlay, State) ->
     VTEPIPStr = mget(<<"vtep_ip">>, VXLan),
     VTEPIP6Str = mget(<<"vtep_ip6">>, VXLan),
     VTEPMac = mget(<<"vtep_mac">>, VXLan),
-    maybe_add_overlay_to_lashup(VTEPIPStr, VTEPMac, AgentSubnet, OverlaySubnet, State),
-    maybe_add_overlay_to_lashup(VTEPIP6Str, VTEPMac, AgentSubnet6, OverlaySubnet6, State).
+    VTEPName = mget(<<"vtep_name">>, VXLan),
+    maybe_add_overlay_to_lashup(VTEPName, VTEPIPStr, VTEPMac, AgentSubnet, OverlaySubnet, State),
+    maybe_add_overlay_to_lashup(VTEPName, VTEPIP6Str, VTEPMac, AgentSubnet6, OverlaySubnet6, State).
 
-maybe_add_overlay_to_lashup(_VTEPIPStr, _VTEPMac, _AgentSubnet, undefined, _State) ->
+maybe_add_overlay_to_lashup(_VTEPName, _VTEPIPStr, _VTEPMac, _AgentSubnet, undefined, _State) ->
     ok;
-maybe_add_overlay_to_lashup(undefined, _VTEPMac, _AgentSubnet, _OverlaySubnet, _State) ->
+maybe_add_overlay_to_lashup(_VTEPName, undefined, _VTEPMac, _AgentSubnet, _OverlaySubnet, _State) ->
     ok;
-maybe_add_overlay_to_lashup(VTEPIPStr, VTEPMac, AgentSubnet, OverlaySubnet, State) ->
+maybe_add_overlay_to_lashup(VTEPName, VTEPIPStr, VTEPMac, AgentSubnet, OverlaySubnet, State) ->
     ParsedSubnet = parse_subnet(OverlaySubnet),
     Key = [navstar, overlay, ParsedSubnet],
     LashupValue = lashup_kv:value(Key),
     case check_subnet(VTEPIPStr, VTEPMac, AgentSubnet, LashupValue, State) of
-        ok ->
-            ok;
+        ok -> ok;
         Updates ->
-            lager:info("Overlay poller updating lashup"),
-            {ok, _} = lashup_kv:request_op(Key, {update, [Updates]})
+            {ok, _} = lashup_kv:request_op(Key, {update, [Updates]}),
+            Info = #{ip => VTEPIPStr, mac => VTEPMac,
+                     agentSubnet => AgentSubnet, overlaySubnet => OverlaySubnet},
+            lager:notice("Overlay network ~s was added, ~p", [VTEPName, Info])
```

Comment:
maybe `Overlay network ~s pushed to lashup, ~p`

Reply:
I'd like to avoid such details like `lashup` in dcos-net logs, those logs will be used by many people and they don't know what `lashup` is.

Reply:
shouldn't be a problem as it is an open source software. If they don't know they would soon know. Or we need to find a better term to replace `added` because its confusing as to where is it is `added` to?

Reply:
Does it sound better?

```
Overlay network configuration was broadcasted: ~s => ~p
```

Reply:
yeah better or maybe gossiped?